### ISSUE TITLE
How to handle katex errors.

### ISSUE THREAD

**Issue by sabadoryo** [lucpotage/vue-katex#30]
How can I handle errors which are throwing to console by katex itself? I want to show this error messages in my vue component directly to users.
Example:
![image](https://user-images.githubusercontent.com/56064241/94366701-c2cdf180-00fb-11eb-8f37-273fce09b693.png)

To render latex I am using:
`<div v-katex:auto="{options}" :key="item.text">.... </div>`

**Comment by tuanalumi** [comment]
Why is this closed? Do you find the solution @sabadoryo ?

**Comment by lucpotage** [comment]
Indeed, I think we can do better. I wanted to update the plugin for Vue 3 and keep compatibility with Vue 2 as well. I will add this feature that is handy. I cannot commit to a date. For specific use of katex, it's also possible to create your own component, this lib being a very light wrapper around the original katex library.
